lib/ReactViews/Story/StoryBuilder.jsx

```diff
@@ -417,15 +417,16 @@ const StoryBuilder = observer(
                       key={`${story.id}`}
                       story={story}
                       sortData={story}
-                      deleteStory={this.removeStory.bind(this, index)}
-                      recaptureStory={this.recaptureScene}
+                      deleteStory={() => this.removeStory(index, story)}
```

Comment:
I feel like I'm missing something obvious but why are all these event handlers wrapped in closures now?

Reply:
ah this is so that the function that's passed in as props has the story bound already. Previously, we had code like `props.deleteStory(props.story)` which doesn't quite make sense to me.

The reason why i didn't do something like `this.removeStory.bind(this, index)` is that react throws up a warning about how it auto-binds this in a performant way which is why i have the anonymous functions.